backendless.d.ts

```diff
@@ -1435,6 +1435,204 @@ declare module Backendless {
         radius: number;
         units: string;
     }
+
+    class UnitOfWorkResult {
+    }
+
+    class OpResultValueReference {
+
+    }
+
+    class OpResult {
+        opResultId: string;
+
+        setOpResultId(opResultId: string): void;
+
+        resolvedTo(index: number, property?: string): OpResultValueReference
+        resolvedTo(property: string): OpResultValueReference
+    }
+
+    enum IsolationLevelEnum {
+        REPEATABLE_READ,
+        READ_COMMITTED,
+        READ_UNCOMMITED,
+        SERIALZABLE
```

Comment:
Is this not a typo? SERIALZABLE
Why not SERIALIZABLE?

Reply:
fixed